composer.json

```diff
@@ -1,11 +1,14 @@
 {
+  "name": "S1SYPHOS/kirby-webp",
+  "type": "kirby-plugin",
```

Comment:
If I understand correctly, this maps the install location to `site/plugins/`, doesn't it? My projects tend to have a different structure, inside root there are `source`, `build`, `package.json` and so on .. so basically the baseDir, the directory containing `site`, would be `build`.

How will this be configured? From the issues already raised about this, I get it that in this case you have to override it. Are you familiar with this? Maybe there's an easier way :+1:

Reply:
Yes it puts all files inside the site/plugins/your-plugin-name folder. I am not sure what you mean with the build and source folder but all your files should be inside a folder that's inside the sites/plugins folder...

Reply:
Nevermind